packages/deployment/src/handlers/default/index.ts

```diff
@@ -1,8 +1,10 @@
 import { CloudFrontRequest, CloudFrontRequestEvent, CloudFrontRequestHandler } from 'aws-lambda';
 import { join } from 'path';
 
+import { RUNTIME_SETTINGS_FILE } from 'constants/handlerPaths';
+
 import { extractDynamicParams, matchParams } from '../../helpers/dynamic';
-import { defaultRuntimeManifest } from '../../types/manifests';
+import { defaultRuntimeSettings } from '../../types/runtimeSettings';
```

Comment:
ca marche les imports absolus ? 
```suggestion
import { defaultRuntimeSettings } from 'types/runtimeSettings';
```